misk/src/main/kotlin/misk/tracing/Trace.kt

```diff
@@ -0,0 +1,11 @@
+package misk.tracing
+
+/**
+ * Annotate a method with @Trace to enable tracing for the specified method.
+ *
+ * NOTE: This annotation is implemented using Guice's method interception capabilities.
+ * As such, methods that can be traced using this annotation are subject to the same limitations
+ * outlined in the <a href="https://github.com/google/guice/wiki/AOP">Guice wiki</a>.
```

Comment:
it sounds like there is some Kotlin issues though, like requiring `open`, since final is the default. it would be nice to document those too

Reply:
Good catch. Meant to document those, but forgot